src/App.tsx

```diff
@@ -15,48 +17,131 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  name = 'name',
+  length = 'length',
+  empty = '',
```

Comment:
enum's property should be started with the capital word 
```suggestion
  Name = 'name',
  Length = 'length',
  Empty = '',
```